pipelines/release.yml

```diff
@@ -1214,10 +1212,9 @@ resources:
 - name: charts
   type: git
   icon: *git-icon
-  version: {ref: ((charts_version))}
   source:
-    uri: https://github.com/helm/charts.git
-    branch: master
+    uri: https://github.com/concourse/concourse-chart.git
```

Comment:
This change may break k8s-topgun and anything else that used this chart repo. We hard-coded a bunch of `stable/concourse` stuff. Let me see where those transition things are at with the repo. 

Then again, this would be a really nice way of telling us where we need to make changes 😄

Reply:
Can we remove this diff from this PR? We have #200 which will make this change and also fix all the scripts this impacts.

Reply:
It should be a no-op regardless of merge order I think?

Reply:
Guess so, no harm then! Will only break k8s stuff anyways which isn't a blocker for releasing.